.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: wc2daTtJVXyxbgbn9C3K4cnGO67TBffSW
```

Comment:
It's not advisable to push this file, as it contains your repo token which should be confidential.
repo token is only useful when you want to report coverage to coveralls directly from your local environment.